content/v2/overview/authentication.md

```diff
@@ -5,18 +5,24 @@ title: API v2 - Overview - Authentication
 # Authentication
 
 SeeClickFix currently supports the following authentication methods for access to API v2.
-1. HTTP Basic Authentication
+1. Token Authentication
```

Comment:
While these are showing up as ordered list in the GitHub markdown preview, they are not translating as such to the Jekyll site.

* GitHub Preview:
<a href="https://github.com/user-attachments/assets/246c2059-346f-4cac-bec3-e7640580fb7e"><img src="https://github.com/user-attachments/assets/246c2059-346f-4cac-bec3-e7640580fb7e" width="600"></a>
* Published:
<a href="https://github.com/user-attachments/assets/79ba88a2-7827-43e5-b3e6-e372de92a46f"><img src="https://github.com/user-attachments/assets/79ba88a2-7827-43e5-b3e6-e372de92a46f"></a>

Reply:
Line break after line 7 should fix this.